Gaijinizer patch/Map071.txt

```diff
@@ -4,117 +4,123 @@
 > CONTEXT: Map071/events/1/pages/0/23/Dialogue/other3
-Are you okay? \\N[1] you? Isn't it a place where
-you feel something wrong?#MTLed
+Are you okay, \\N[1]-kun? 
+
+Does your body feel strange anywhere?#Ron
 > END STRING
 
 > BEGIN STRING
 い、違和感なら、今めっちゃ感じてるけど……
 
 \\{ていうか、あんた誰！？
 > CONTEXT: Map071/events/1/pages/0/27/Dialogue/kazuya
-No, if you feel strange, I feel really now ...
-\\{I mean, everyone! ?#MTLed
+I- if we're talking strange, this is pretty
+strange what's going on here right now...
+
+\\{I mean, who the hell are you?!#Ron
 > END STRING
 
 > BEGIN STRING
 ワシか？
 
 ワシは結（ゆい）。カラステングの発明家じゃ。
 > CONTEXT: Map071/events/1/pages/0/31/Dialogue/other3
-Is it an eagle? The eagle is Yui. You are the
-inventor of Karasu Tengu.#MTLed
+Me? The name's Yui, the Karasu Tengu inventor.#Ron
 > END STRING
 
 > BEGIN STRING
 お前さんを転送装置でここに移動させて
 
  あの悪魔から逃がしたのは、何を隠そうワシなんよ！
 > CONTEXT: Map071/events/1/pages/0/35/Dialogue/other3
-Move you here with the transfer device
-What escaped from that demon is what an eagle is
-hiding!#MTLed
+I transferred you here with my transfer device and
+now I'm hiding you here, I am!#RonIffy
 > END STRING
 
 > BEGIN STRING
 はぁ……
 
 て、転送装置………？
 > CONTEXT: Map071/events/1/pages/0/39/Dialogue/kazuya
-Ha..., Transfer device......?#MTLed
+Huh...
+
+T- transfer device...?#Ron
 > END STRING
 
 > BEGIN STRING
 結…………カラステング………発明家……。
 
 もしやとは思うが………
 > CONTEXT: Map071/events/1/pages/0/45/Dialogue/yuuma
-.......... Karasu Tengu ..... Inventors ....
-I think it's a shame .........#MTLed
+Yui... the Karasu Tengu... inventor...
+
+To think there was one...#RonIffy
 > END STRING
 
 > BEGIN STRING
 ………？
 
 知っているのか？ユウマ。
 > CONTEXT: Map071/events/1/pages/0/49/Dialogue/kazuya
-.........? Do you know? Yuuma.#MTLed
+...?
+
+Do you know her, Yuuma?#Ron
 > END STRING
 
 > BEGIN STRING
 確か………『悪魔弾』の開発の第一人者が
 
  カラステングの一族の、結という悪魔だったと聞いている。
 > CONTEXT: Map071/events/1/pages/0/53/Dialogue/yuuma
-Certainly...... the first person in the
-development of "Demon rounds". 
-I heard that it was a unity demon of the Karasu
-Tengu clan.#MTLed
+If I remember correctly... I'd heard that a demon
+called Yui from the Karasu Tengu clan spearheaded
+the development of "Demon Rounds".#Ron
 > END STRING
 
 > BEGIN STRING
 『旧世代』の銃弾の規格に合わせた火薬の調合や
 
 魔具の軍用兵器化にも大きく貢献したとか……
 > CONTEXT: Map071/events/1/pages/0/57/Dialogue/yuuma
-Mix of explosives to the standard of "outdated"
-bullets. He also contributed greatly to the
-transformation of magic tools into military
-weapons ...#MTLed
+She contributed greatly by doing things like
+combining standard "outdated" ammo with low-yield
+explosives, and converting magic tools into
+military-grade weapons.#Ron
 > END STRING
 
 > BEGIN STRING
 そうそう！今\\N[1]君が使っとる銃の弾も
 
  ワシが最初に発明したもんなんやで！！
 > CONTEXT: Map071/events/1/pages/0/61/Dialogue/other3
-Yes Yes! The gun bullet that \\N[1] you use now
-By what the eagle invented first! !#MTLed
+That's right! Those bullets you've been using, 
+\\N[1], I invented those first!!#Ron
```

Comment:
There's a lot of possible ways to parse this:

![image](https://user-images.githubusercontent.com/48231102/69084389-c25bc900-0a09-11ea-8993-308b0ea6b23b.png)

GT says that Demon Rounds are the first thing she invented. Other translators say that she was the first one to invent Demon Rounds. I'm assuming the latter is the proper way to read this?

Reply:
Yeah it's a bit ambigious. It's _probably_ not the first thing she invented, just because I'd expect to see the 最初 as a modifying adjective with の after the end, rather than being used as an adverb with に. But then it doesn't make much sense that she was the first one to invent them because it's implied she (a demon) is the one to thank for demon rounds instead of whatever human independantly also invented them.

Hmmm, now I've talked myself into the other interpretation (that they were the first thing she invented)